src/components/Slider/Slider.test.tsx

```diff
@@ -15,18 +14,16 @@ describe("Slider", () => {
         value={5}
       />
     );
-    expect(wrapper.find("input[type='number']").exists()).toBe(false);
+    expect(screen.queryByRole("textbox")).not.toBeInTheDocument();
   });
 
   it("can render with an additional number input", () => {
-    const wrapper = shallow(
+    render(
       <Slider max={10} min={0} onChange={jest.fn()} showInput value={5} />
     );
-    expect(wrapper.find("input[type='number']").exists()).toBe(true);
+    expect(screen.getByRole("spinbutton")).toBeInTheDocument();
```

Comment:
Why are these tests looking for different roles? Shouldn't they both be looking for either spinbutton or textbox?

Reply:
Yeah good catch! The dangers of `.not.toBeInTheDocument()`! Fixed.